docs/balsamic_map.md

```diff
@@ -1,84 +1,49 @@
+| cram, umi-normal-cram                                                 | False       | umi-cram, normal                      | deliver               |
+| cram, umi-normal-cram-index                                           | False       | umi-cram-index, normal                | deliver               |
+| vcf-svdb, research-vcf-svdb                                           | True        | svdb, vcf-sv-research                 | deliver               |
+| vcf-svdb, research-vcf-svdb-index                                     | True        | svdb, vcf-sv-research-index           | deliver               |
+| vcf-pass-svdb, clinical-vcf-pass-svdb                                 | True        | svdb, vcf-sv-clinical                 | deliver, scout        |
+| vcf-pass-svdb, clinical-vcf-pass-svdb-index                           | True        | svdb, vcf-sv-clinical-index           | deliver, scout        |
+| snv, vcf-all, dnascope, tumor, annotated-germline-vcf-all             | True        | dnascope, germline, tumor, vcf        | deliver               |
+| annotated-germline-vcf-all-index, snv, vcf-all, dnascope, tumor       | True        | dnascope, germline, tumor, vcf-index  | deliver               |
+| snv, vcf-all, normal, dnascope, annotated-germline-vcf-all            | False       | dnascope, germline, normal, vcf       | deliver               |
+| annotated-germline-vcf-all-index, snv, vcf-all, normal, dnascope      | False       | dnascope, germline, normal, vcf-index | deliver               |
+| manta-germline, vcf-all, sv, tumor, annotated-germline-vcf-all        | True        | manta, germline, tumor, vcf           | deliver               |
+| annotated-germline-vcf-all-index, manta-germline, vcf-all, sv, tumor  | True        | manta, germline, tumor, vcf-index     | deliver               |
+| manta-germline, vcf-all, normal, sv, annotated-germline-vcf-all       | False       | manta, germline, normal, vcf          | deliver               |
+| annotated-germline-vcf-all-index, manta-germline, vcf-all, normal, sv | False       | manta, germline, normal, vcf-index    | deliver               |
+| vcf-tnscope, research-vcf-tnscope                                     | False       | tnscope, vcf-snv-research             | deliver               |
+| vcf-tnscope, research-vcf-tnscope-index                               | False       | tnscope, vcf-snv-research-index       | deliver               |
+| snv, vcf-pass-tnscope, clinical-vcf-pass-tnscope                      | False       | tnscope, vcf-snv-clinical             | deliver, scout        |
+| snv, vcf-pass-tnscope, clinical-vcf-pass-tnscope-index                | False       | tnscope, vcf-snv-clinical-index       | deliver, scout        |
+| tmb, snv, stat-somatic-tmb, tnscope                                   | False       | tnscope, tmb                          | deliver               |
```

Comment:
```suggestion
| tmb, snv, stat-somatic-tmb, tnscope                                   | False       | tnscope, tmb                          | deliver               |
```
Include or exclude for now?

Reply:
Forgot to update this, I will do it when we finish defining the `tags`/`used by` fields.

Reply:
[3254dd5](https://github.com/Clinical-Genomics/hermes/pull/56/commits/3254dd5a0ebcb9b7acee3c01fdb4a2858bdf8f5c)